backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/SysMLElementSerializer.java

```diff
@@ -208,6 +226,108 @@ public String casePartUsage(PartUsage partUsage) {
         return this.appendDefaultUsage(this.newAppender(), partUsage).toString();
     }
 
+    @Override
+    public String caseActionUsage(ActionUsage actionUsage) {
+
```

Comment:
Please do not let empty lines on first lines of methods, here and everywhere else.

Reply:
Done